File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/oracle/BinaryTypes.kt
package dev.typr.foundationskt.docs.oracle

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class BinaryTypes {
    //start
    val rawType: OracleType<ByteArray> = OracleTypes.raw
    val raw100: OracleType<ByteArray> = OracleTypes.raw(100)  // RAW(100)

    // Non-empty variant
    val rawNonEmpty: OracleType<NonEmptyBlob> = OracleTypes.rawNonEmpty(100)
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/duckdb/BinaryTypes.scala
package dev.typr.foundationssc.docs.duckdb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object BinaryTypes:
  //start
  val blobType: DuckDbType[Array[Byte]] = DuckDbTypes.blob
  //stop
